src/ui/public/visualize/loader/vis.js

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
this Vis object is mostly copied out of `ui/vis/vis.js` ...
we need that object still as it is until we migrate to new saved object format, but we will need this new Vis object for passing into renderers (doesn't hold any aggs, schemas, indexPatterns, ... )